docs/getting-started/install-zowe-sdks.md

```diff
@@ -86,14 +100,60 @@ npm install core-for-zowe-sdk.tgz
 
 ### Python SDK *technical preview*
 
-If you install Python SDK packages from the online registry, the required dependencies are installed automatically.
+If you install Python SDK packages from the [online registry](#installing-an-sdk-from-an-online-registry), the required dependencies are installed automatically. **[should we add this sentence to the Kotlin and Java sections above?]**
 
 If you download the Python SDK packages from Zowe.org, the downloaded folder contains dependencies that you must install manually. Extract the `WHL` files from the folder, copy the files to your project, and issue the following command for each dependency:
 
 ```
 pip install <fileName>.whl
 ```
 
-## Getting started
+## Installing a Zowe Client SDK
+
+To get started, import the SDK package to your project. You can pull the packages from an online registry, or download the packages from Zowe.org to install locally.
+
+### Installing an SDK from an online registry
+
+Pull the packages from an online registry such as npm or PyPi:
+
+1. In command-line window, navigate to your project directory. Issue the following command to install a package from the registry:
+
+   - To import a Node.js package: `npm install <PackageName>`
+   - To import a Python package: `pip install <PackageName>`
+
+      - `<packageName>`
+      The name of the SDK package that you want to install, such as `zos-files-for-zowe-sdk`.
+
+    The package is installed. Node packages are defined in `package.json` in your project. Python packages are installed by default to `$PYTHONPATH/Lib/site-packages` (Linux) or to the Python folder in your local `/AppData` folder (Windows). **[what about java and kotlin?]**
```

Comment:
@adam-wolfe - please respond

Reply:
For the Java and Kotlin SDKs, Maven puts libraries in the `~/.m2/repository` directory and Gradle puts libraries in the `~/.gradle/caches/modules-2/files-2.1` directory where ~ represents the path to the user's home directory.